data/ignition/systemd/units/start-cluster-installation.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Service that starts cluster installation
+Wants=network-online.target create-cluster-and-infra-env.service
+After=network-online.target create-cluster-and-infra-env.service
+ConditionPathExists=/etc/assisted-service/node0
```

Comment:
We should think about what we want our trigger to be for determining we are in the automated flow (and hence running this unit) vs. the future interactive flow. I assume it's the existence of some directory of config files.